### PR TITLE
fix: remove market price bug on currency change

### PR DESCRIPTION
**Pull request by berteotti**
# Summary

Remove market price bug on currency change.

Previously the limit price didn't match the input labels and market price difference when changing currency.

  # To Test

1. Go to limit orders
2. Change to a new currency
3. Immediately swap limit order type (from sell to buy or vice-versa)   
- [ ] Everything should be consistent on the UI

### CODE REVIEW

File: src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/OrderLimitPriceField.tsx [hunk 1]
```diff
@@ -141,7 +141,12 @@ export function OrderLimitPriceField({ id }: OrderLimitPriceFieldProps) {
       }
     }
   }
-
+  console.log('toggle out', {
+    limitOrderKind: limitOrder.kind,
+    formattedLimitPrice,
+    baseTokenAmount,
+    quoteTokenAmount,
+  })
```
Comment (adamazad) [inline src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/OrderLimitPriceField.tsx:149]: If you want to debug, I'm totally ok with `debug`

**Review by adamazad — APPROVED** [review]

**Review by Mi-Lan — APPROVED** [review]

**Comment by 0xVenky** [comment]
Works perfect!

File: src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/OrderLimitPriceField.tsx (hunk 1 above)
Comment (bejzik8) [reply src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/OrderLimitPriceField.tsx:149]: Agree!

**Review by bejzik8 — COMMENTED** [review]
Everything looks good to me Leo! Great job! 👏🏼

File: src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/OrderLimitPriceField.tsx (hunk 1 above)
Comment (berteotti) [reply src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/OrderLimitPriceField.tsx:149]: Forgot about this, thanks. I will remove

**Pull request merged**